toolkit/themes/shared/design-system/design-tokens.json

```diff
@@ -55,6 +58,420 @@
+            }
+          },
+          "disabled": {
+            "value": {
+              "default": "{button.background.color.destructive.@base}",
+              "forcedColors": "{button.background.color.primary.disabled}"
+            }
+          },
+          "hover": {
+            "value": {
+              "light": "{color.red.60}",
+              "dark": "{color.red.10}",
+              "forcedColors": "{button.background.color.primary.hover}"
+            }
+          }
+        },
+        "ghost": {
+          "@base": {
+            "value": {
+              "default": "transparent",
+              "prefersContrast": "{button.background.color.@base}",
+              "forcedColors": "{button.background.color.@base}"
+            }
+          },
+          "active": {
+            "value": "{button.background.color.active}"
+          },
+          "disabled": {
+            "value": {
+              "default": "{button.background.color.ghost.@base}",
+              "forcedColors": "{button.background.color.disabled}"
+            }
+          },
+          "hover": {
+            "value": "{button.background.color.hover}"
+          }
+        }
+      }
+    },
+    "border": {
+      "@base": {
+        "value": "{border.width} solid {button.border.color.@base}"
+      },
+      "color": {
+        "@base": {
+          "value": {
+            "default": "transparent",
+            "prefersContrast": "{button.text.color.@base}",
+            "forcedColors": "{border.interactive.color.@base}"
+          }
+        },
+        "active": {
+          "value": {
+            "default": "{button.border.color.@base}",
+            "forcedColors": "{border.interactive.color.active}"
+          }
+        },
+        "destructive": {
+          "@base": {
+            "value": {
+              "default": "transparent",
+              "forcedColors": "{button.border.color.primary.@base}"
+            }
+          },
+          "active": {
+            "value": {
+              "default": "{button.border.color.destructive.@base}",
+              "forcedColors": "{button.border.color.primary.active}"
+            }
+          },
+          "disabled": {
+            "value": {
+              "default": "{button.border.color.destructive.@base}",
+              "forcedColors": "{button.border.color.primary.disabled}"
+            }
+          },
+          "hover": {
+            "value": {
+              "default": "{button.border.color.destructive.@base}",
+              "forcedColors": "{button.border.color.primary.hover}"
+            }
+          }
+        },
+        "disabled": {
+          "value": {
+            "default": "{button.border.color.@base}",
+            "forcedColors": "{border.interactive.color.disabled}"
+          }
+        },
+        "ghost": {
+          "@base": {
+            "value": {
+              "default": "{button.border.color.@base}",
+              "forcedColors": "{button.border.color.@base}"
```

Comment:
Do we need both of these or can this be cleaned up? (same thing happens with "active" and "hover" here - it seems like we shouldn't need to redefine this in forced colors mode)